lib/interpolators.js

```diff
@@ -0,0 +1,173 @@
+// Return the value of an opportunity density grid cell.
+// Range check and return 0 for cells outside the grid.
+// TODO This should really be in the Grid class, and grids should have a default value for empty cells.
+function getGridValue (grid, gx, gy) {
```

Comment:
In general we should use Javadoc-style comments (`/** . . . */`) to describe classes and methods.